### DISCUSSION TITLE
QFieldCloud Sync not working

### DISCUSSION THREAD

**Discussion by martinfischer8** [opengisch/QField#5707] [Bugs]
Last week, September 24th, I was able to sync to my smartphone without problems. After doing some field work I'd like to push the changes. Until today this is not working. When I press the sync or the push button on my phone, nothing happens, no error message or anything besides a short animation of the button.

There were no changes to the project. What I noticed is that the app was updated to Version 3.4.2 - Ebo on September 26th.

So I wonder whether is a problem with the app itself or the update went wrong. I'd like to uninstall the app and re-install. But how can I do that without loosing data? Anything else I can do?

I'm using QField on Android.

Greetings,
Martin

**Comment by BoswachterMarc** [comment]
See https://github.com/opengisch/QField/issues/5708

**Comment by BoswachterMarc** [comment]
> Anything else I can do?

You can export the data.gpkg file (go to project folder and tap on the three dots)  in your QField and import/add/copy the missing data in QGis. But at this point, I would wait for a response/solution from the developers.

**Reply by martinfischer8** [reply]
Thank you. I think being patient and wait for a upgraded app will be the way for me.

**Reply by m-kuhn** [reply]
@nirvn does the new release include a fix?

**Reply by nirvn** [reply]
@m-kuhn , that's correct.

**Comment by martinfischer8** [comment]
Problem is solved with Version 3.4.3

**Reply by nirvn** [reply]
@martinfischer8 , is solved or is _not_ solved?

**Reply by BoswachterMarc** [reply]
In my opinion it IS solved ;-)

**Reply by nirvn** [reply]
@BoswachterMarc , I sure hope so ;)
